ocean_lib/ocean/deploy.py

```diff
@@ -9,6 +9,10 @@
 import os
 import sys
 
+# Setup ocean_lib.enforce_typing_shim before importing anything that uses it
+from ocean_lib.enforce_typing_shim import setup_enforce_typing_shim
```

Comment:
I think same here, with isort. Goes without telling everywhere this syntax is used. I'm not going to comment on each one, not to overcrowd the review.